### ISSUE TITLE
@org_tensorflow//tensorflow/lite/schema components visibility

### ISSUE THREAD

**Issue by oberluz** [tensorflow/tensorflow#62323]
### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

tf 2.13+

### Custom code

No

### Bazel version

6.1.0

### Current behavior?

Hi,

I am updating https://github.com/google-coral/pycoral to support Tensorflow 2.13+. This repo used to be able to reference and use the following bazel libs:

@org_tensorflow//tensorflow/lite/schema:schema_conversion_utils
@org_tensorflow//tensorflow/lite/schema:schema_utils

But the visibility of these components was changed from v2.2.0 and is now ":utils_friends"

Could these be reverted to public so we can continue to use them or else suggest how I might use them as they currently stand?

Many thanks

### Standalone code to reproduce the issue

```shell
.
```

### Relevant log output

```shell
ERROR: /workspace/coral/learn/BUILD:21:11: in cc_library rule //coral/learn:utils: target '@org_tensorflow//tensorflow/lite/schema:schema_conversion_utils' is not visible from target '//coral/learn:utils'. Check the visibility declaration of the former target if you think the dependency is legitimate
ERROR: /workspace/coral/learn/BUILD:21:11: in cc_library rule //coral/learn:utils: target '@org_tensorflow//tensorflow/lite/schema:schema_utils' is not visible from target '//coral/learn:utils'. Check the visibility declaration of the former target if you think the dependency is legitimate
ERROR: /workspace/coral/learn/BUILD:21:11: Analysis of target '//coral/learn:utils' failed
```

**Comment by pkgoogle** [comment]
Hi @sirakiin, can you please take a look?

**Comment by oberluz** [comment]
Yes, it seems it always was. I think I've  resolved my issue: I realised I was using bazel 6.1.0 instead of 5.3.0 which is the version used in v2.13.0.

Thank you for looking into this.
